tools/src/commands/RunStories.ts

```diff
@@ -0,0 +1,241 @@
+import spawnAsync from '@expo/spawn-async';
+import { IosPlist } from '@expo/xdl';
+import fs from 'fs';
+import inquirer from 'inquirer';
+import path from 'path';
+import { podInstallAsync } from '../CocoaPods';
+
+import { runExpoCliAsync } from '../ExpoCLI';
+import Logger from '../Logger';
+
+type Action = {
+  platform: 'android' | 'ios' | 'web';
+  rebuild: boolean;
+  clearCache: boolean;
+};
+
+async function action(packageName: string, options: Action) {
+  let { platform, rebuild = false, clearCache = false } = options;
```

Comment:
As Julius Cesar showed us long time ago: `veni, vidi, inline` 👑 
```suggestion
async function action(packageName: string, { platform, rebuild = false, clearCache = false }: Action) {
```